tests/test_packages/test_connections/test_p2p_libp2p_client/test_aea_cli.py

```diff
@@ -19,52 +19,54 @@
 # ------------------------------------------------------------------------------
 
 """This test module contains AEA cli tests for Libp2p tcp client connection."""
+
 import json
 
-from aea.helpers.base import CertRequest
 from aea.multiplexer import Multiplexer
 from aea.test_tools.test_cases import AEATestCaseEmpty
 
 from packages.valory.connections import p2p_libp2p_client
-from packages.valory.connections.p2p_libp2p.consts import (
-    LIBP2P_CERT_NOT_AFTER,
-    LIBP2P_CERT_NOT_BEFORE,
-)
 from packages.valory.connections.p2p_libp2p_client.connection import PUBLIC_ID
 
 from tests.conftest import DEFAULT_LEDGER, LOCAL_HOST
 from tests.test_packages.test_connections.test_p2p_libp2p.base import (
     _make_libp2p_connection,
     libp2p_log_on_failure_all,
+    make_cert_request,
     ports,
 )
 
 
-p2p_libp2p_client_path = f"vendor.{p2p_libp2p_client.__name__.split('.', 1)[-1]}"
 DEFAULT_HOST = LOCAL_HOST.hostname
 DEFAULT_CLIENTS_PER_NODE = 4
 DEFAULT_LAUNCH_TIMEOUT = 10
 
 
 @libp2p_log_on_failure_all
 class TestP2PLibp2pClientConnectionAEARunning(AEATestCaseEmpty):
-    """Test AEA with p2p_libp2p_client connection is correctly run"""
+    """Test AEA with client connection is correctly run"""
+
+    conn_path = f"vendor.{p2p_libp2p_client.__name__.split('.', 1)[-1]}"
+    public_id = str(PUBLIC_ID)
+    port = next(ports)
+    uri = f"{DEFAULT_HOST}:{port}"
+    kwargs = dict(
+        delegate_host=DEFAULT_HOST,
+        delegate_port=port,
+        delegate=True,
+    )
```

Comment:
defined here to allow for easy override in mailbox tests, allowing for inheritance to be used